memory/ndctl.py

```diff
@@ -38,21 +38,29 @@ class NdctlTest(Test):
 
     """
 
-    def get_json(self, option=''):
+    def get_json(self, short_opt='', long_opt=''):
         """
         Get the json of each provided options
 
         return: By default returns entire detail of namespaces
         """
+        if short_opt and long_opt:
```

Comment:
This check is unnecessary, since the function param is all static.
Also, [] is returned in 2 different cases: option mistake, and value error.
Also, it is better to add this behaviour to docstring

Reply:
Right, anyway only the test uses it so will remove the check or document it.